org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -754,12 +758,30 @@ SsoSynapseAdmin:
   OrganizationBindings:
     TargetBinding:
       OrganizationalUnit:
-        - !Ref SynapseOU
+        - !Ref SynapseProdAccount
```

Comment:
I think we should probably replace the above `synapseAdminGroup:      #JC aws-synapse-admins` with a more specific `synapseProdAdminGroup`?  it can be done in a separate PR but I just wanted to see if that was in your plans.

Reply:
Yes, that's the plan.